database/migrate/migrations/00017_add_blob_meta_data.sql

```diff
@@ -0,0 +1,27 @@
+-- +goose Up
+-- +goose StatementBegin
+
+ALTER TABLE chunk
+ADD COLUMN crc_max INTEGER NOT NULL,
```

Comment:
Will `NOT NULL` work here (for `crc_max` and `data_hash`)? Since the DB will already contain `NULL` values for these columns.